metals/src/main/scala/scala/meta/internal/metals/DefinitionProvider.scala

```diff
@@ -89,7 +89,7 @@ final class DefinitionProvider(
     val occurrence = for {
       queryPosition <- snapshotPosition.toPosition(dirtyPosition.getPosition)
       occurrence <- snapshot.occurrences
-        .find(_.encloses(queryPosition, includeLastCharacter))
+        .find(_.encloses(queryPosition, config.includeLastCharacter))
```

Comment:
we can set it to true here ande remove from config since we don't plan to have different behaviour per client.